packages/flutter_test/lib/src/binding.dart

```diff
@@ -84,7 +84,6 @@ const Size _kDefaultTestViewportSize = const Size(800.0, 600.0);
 abstract class TestWidgetsFlutterBinding extends BindingBase
```

Comment:
maybe add to the last paragraph here, saying that the "also, the ServicesBinding is not included" or some such?